linalg/blockmatrix.cpp

```diff
@@ -15,7 +15,7 @@
 #include "sparsemat.hpp"
 #include "blockvector.hpp"
 #include "blockmatrix.hpp"
-
+#include <vector>
```

Comment:
There should be an empty line after the last include.

Reply:
fixed